### ISSUE TITLE
Datepicker label is missing the "for" attribute

### ISSUE THREAD

**Issue by snapsnapturtle** [freenowtech/wave#24]
Browsers cannot link the label to the input to provide an accessible experience

**Comment by duranmla** [comment]
We may want to understand how other design systems deal with a11y on DatePicker, not only the "for" attribute but the actual complemented implementation of an "accessible date picker"

**Comment by nlopin** [comment]
I believe that was fixed with #86
